python/src/main/resources/runtime/src/rosetta/runtime/utils.py

```diff
@@ -540,7 +607,7 @@ def get_only_element(collection):
 
 def flatten_list(nested_list):
     '''flattens the list of lists (no-recursively)'''
-    return [item for sublist in nested_list for item in sublist]
+    return [item for sublist in _to_list(nested_list) for item in _to_list(sublist)]
 
 
 def rosetta_filter(items, filter_func, item_name='item'):
```

Comment:
The parameter `item_name` should be removed. The lambda func is invoked with iem.